themes/theme-b2c-tailwind/src/molecules/quantity-selector.css

```
@@ -0,0 +1,41 @@
+[data-store-quantity-selector-input] {
+}
+
+[data-store-quantity-selector-button] {
+}
+
+[data-store-icon] {
+}
+
+[data-store-quantity-selector] {
+}
+
+.quantitySelector [data-store-icon] {
```

We don't need this class, because we can use the data attribute, example:

```suggestion
[data-store-quantity-selector] [data-store-icon] 
```